pkg/tsdb/elasticsearch/data_query.go

```diff
@@ -493,3 +497,9 @@ func stringToIntWithDefaultValue(valueStr string, defaultValue int) int {
 	}
 	return value
 }
+
+func addPluginErrorToResponse(refID string, response *backend.QueryDataResponse, err error) *backend.QueryDataResponse {
```

Comment:
[pr](https://github.com/grafana/grafana-plugin-sdk-go/pull/795/files#diff-cf82f3baeb1eb237f38b3190af31d740586f6fbf3db8180317719dee4fa4c184R79) to add this function to sdk